app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -1,94 +1,72 @@
+      <% end %>
     <% end %>
-  <% end %>
 
-  <%= render(SummaryCardComponent.new(rows: course_choice_rows(application_choice), editable: @editable)) do %>
-    <%= render(SummaryCardHeaderComponent.new(title: application_choice.current_course.provider.name, heading_level: @heading_level)) do %>
-      <div class="app-summary-card__actions">
-        <% if application_choice.offer? %>
-          <%= govuk_link_to candidate_interface_offer_path(application_choice.id), data: { action: :respond } do %>
-            <%= t('application_form.courses.view_and_respond_to_offer') %>
-            <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
+    <%= render(SummaryCardComponent.new(rows: course_choice_rows(application_choice), editable: @editable)) do %>
+      <%= render(SummaryCardHeaderComponent.new(title: application_choice.current_course.provider.name, heading_level: @heading_level)) do %>
+        <div class="app-summary-card__actions">
+          <% if application_choice.offer? %>
+            <%= govuk_link_to candidate_interface_offer_path(application_choice.id), data: { action: :respond } do %>
+              <%= t('application_form.courses.view_and_respond_to_offer') %>
+              <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
+            <% end %>
+          <% elsif withdrawable?(application_choice) %>
+            <%= govuk_link_to candidate_interface_withdraw_path(application_choice.id), data: { action: :withdraw } do %>
+              <%= t('application_form.courses.withdraw') %>
+              <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
+            <% end %>
+          <% elsif @editable %>
+            <%= govuk_link_to candidate_interface_confirm_destroy_course_choice_path(application_choice.id), data: { action: :delete } do %>
+              <%= t('application_form.courses.delete') %>
+              <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
+            <% end %>
           <% end %>
-        <% elsif withdrawable?(application_choice) %>
-          <%= govuk_link_to candidate_interface_withdraw_path(application_choice.id), data: { action: :withdraw } do %>
-            <%= t('application_form.courses.withdraw') %>
-            <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
-          <% end %>
-        <% elsif @editable %>
-          <%= govuk_link_to candidate_interface_confirm_destroy_course_choice_path(application_choice.id), data: { action: :delete } do %>
-            <%= t('application_form.courses.delete') %>
-            <span class="govuk-visually-hidden"> <%= application_choice.current_course.name_and_code %></span>
-          <% end %>
-        <% end %>
-      </div>
+        </div>
+      <% end %>
     <% end %>
-  <% end %>
   </div>
 <% end %>
 
```

Comment:
It looks much better! But I do wonder if we should revisit on how structure this into more components? @carlosmartinez wdyt?